Add early years teacher degree apprenticeship (#4488)
DFE-Digital/get-into-teaching-app: app/views/content/life-as-a-teacher/age-groups-and-specialisms/early-years-teachers.md

      - Cost: "Up to $earlyyears_fees_undergrad$ per year"
        Salary: "No"
        Length of training: "3 or 4 years"
        Qualification: "Early years teacher status and a bachelor’s degree"
  EY-self-funded:
    name: feature_table
    arguments:
      - Cost: "This could be around $earlyyears_fees_selffundedlow$ or $earlyyears_fees_selffundedhigh$ depending on the provider"
        Salary: "No"
        Length of training: "3 months"
        Qualification: "Early years teacher status"
  EY-apprenticeship:
    name: feature_table
    arguments:

>       - Cost: "Your course fees of up to $earlyyears_fees_apprenticeship$ are covered through the employer’s apprenticeship levy payments or co-funded with the government contributing 95% of the cost and employers contributing 5% of the cost. You do not need to do anything to access this funding."

Reviewer: ```suggestion
      - Cost: "Your course fees of up to $earlyyears_fees_apprenticeship$ are covered by the Department for Education. You do not need to do anything to access this funding."
```

I think this is probably too much detail for candidates - they just need to know they are not paying?

NB I've suggested saying 'covered by DfE' because I think that would cover both the levy and the co-funding options (and it is also consistent with other routes) - but I don't know this for sure so this need checking!

>       - Cost: "Your course fees of up to $earlyyears_fees_apprenticeship$ are covered through the employer’s apprenticeship levy payments or co-funded with the government contributing 95% of the cost and employers contributing 5% of the cost. You do not need to do anything to access this funding."

Reviewer: For other fees on this page, we say per year - worth checking with the team as to whether we can do the same here for consistency? Otherwise the fee looks huge in comparison!

        Salary: "Provided by employer"
        Length of training: "3 years"

>         Qualification: "Early years teacher status, bachelor’s degree, and level 6 early years teacher apprenticeship"

Reviewer: ```suggestion
        Qualification: "Early years teacher status, a bachelor’s degree, and level 6 early years teacher apprenticeship"
```

---

The early years are critical in children’s development. As an early years teacher you’ll work with young children up to the age of 5. You’ll use your knowledge and skills to help young children enjoy high standards of teaching and open their minds to new ideas every day.

To teach in a nursery or early years setting, you'll need early years teacher status (EYTS). EYTS is also recognised in academies, free schools and independent schools. You can achieve EYTS through either early years initial teacher training (EYITT) or an early years teacher degree apprenticeship (EYTDA).

If you want to teach in a primary school, including in a reception class, then you'll need to achieve qualified teacher status (QTS). [Find out more about training to teach primary pupils](/life-as-a-teacher/age-groups-and-specialisms/primary).

If you're interested in exploring other opportunities to work with pre-school children, you can [find out more about a range of careers in early years education](https://earlyyearscareers.campaign.gov.uk/?utm_source=Website&utm_medium=GiT). 

## What you need to become an early years teacher

> There are several ways to train to become an early years teacher. These routes include undergraduate degree training, an early years teacher degree apprenticeship, or postgraduate options if you already have a degree. All of these routes lead to early years teacher status (EYTS).

Reviewer: @emmaachesongray `include` makes it sound like there are other routes ( I presume there aren't!)?

Would it be better to say something like 'You could do......' (or better worded!) rather than 'These routes include.....'?

> To do postgraduate training, you’ll need a bachelor’s degree or an equivalent, for example, a vocational or professional qualification, or equivalent qualification you gained outside the UK.

Reviewer: @emmaachesongray I wonder if it makes more sense to put this postgrad para after the GCSEs (I know this is in prod already)

ie for all routes you need GCSEs, for postgrad you'll **also** need a degree

(if that is how it works!)

---

The early years are critical in children’s development. As an early years teacher you’ll work with young children up to the age of 5. You’ll use your knowledge and skills to help young children enjoy high standards of teaching and open their minds to new ideas every day.

To teach in a nursery or early years setting, you'll need early years teacher status (EYTS). EYTS is also recognised in academies, free schools and independent schools. You can achieve EYTS through either early years initial teacher training (EYITT) or an early years teacher degree apprenticeship (EYTDA).

If you want to teach in a primary school, including in a reception class, then you'll need to achieve qualified teacher status (QTS). [Find out more about training to teach primary pupils](/life-as-a-teacher/age-groups-and-specialisms/primary).

If you're interested in exploring other opportunities to work with pre-school children, you can [find out more about a range of careers in early years education](https://earlyyearscareers.campaign.gov.uk/?utm_source=Website&utm_medium=GiT). 

## What you need to become an early years teacher

> There are several ways to train to become an early years teacher. You can do  undergraduate degree training, an early years teacher degree apprenticeship, or there are postgraduate options if you already have a degree. All of these routes lead to early years teacher status (EYTS).

Reviewer: ```suggestion
There are several ways to train to become an early years teacher. You can do undergraduate degree training, an early years teacher degree apprenticeship, or there are postgraduate options if you already have a degree. All of these routes lead to early years teacher status (EYTS).
```

$EY-graduate-employment$

### Postgraduate assessment only training

This self-funded route is ideal for graduates with early years experience. If you already meet the [early years teachers’ standards](https://www.gov.uk/government/publications/early-years-teachers-standards) you may be able to gain EYTS without the need for further training.

$EY-self-funded$

### Undergraduate training
This route is for people who want to study for a full-time bachelor’s degree in an early-childhood related subject leading to EYTS.

$EY-undergraduate$

Undergraduate tuition fee loans are available from [Student Finance England (SFE)](https://www.gov.uk/student-finance).

> ### Early years teacher degree apprenticeship (EYTDA)

Author: ```suggestion
### Early years teacher degree apprenticeship
```

$EY-graduate-employment$

### Postgraduate assessment only training

This self-funded route is ideal for graduates with early years experience. If you already meet the [early years teachers’ standards](https://www.gov.uk/government/publications/early-years-teachers-standards) you may be able to gain EYTS without the need for further training.

$EY-self-funded$

### Undergraduate training
This route is for people who want to study for a full-time bachelor’s degree in an early-childhood related subject leading to EYTS.

$EY-undergraduate$

Undergraduate tuition fee loans are available from [Student Finance England (SFE)](https://www.gov.uk/student-finance).

### How to apply for early years initial teacher training

To apply for an early years initial teacher training route, contact accredited early years training providers directly.

<a href="https://www.gov.uk/government/publications/early-years-initial-teacher-training-itt-providers-and-school-direct-early-years-lead-organisations/early-years-initial-teacher-training-itt-providers-and-school-direct-lead-organisations" class="button">Find early years training providers</a>

> ## Early years teacher degree apprenticeship (EYTDA)

Reviewer: ```suggestion
## Early years apprenticeship route
```
Would it make more sense to change the h2 to be slightly more generic, then add an h3 of 'Early years teacher degree apprenticeship (EYTDA)' to be consistent with the ITT routes layout? Or is this adding too much verbosity (quite possibly!)?

Happy to go with whatever you feel best!

Author: I havent added a new H2/H3 as it felt a bit overloaded to have just 1 H3, but have updated the H2 title so it mentions 'route' and is a bit more joined up to the EY ITT H2.

> ### Early years teacher degree apprenticeship (EYTDA) route

Author: ```suggestion
## Early years teacher degree apprenticeship (EYTDA) route
```
